src/components/WizardFooter/WizardFooter.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import { ChevronLeft } from '@amsterdam/asc-assets';
+import {
+	CancelButtonStyle,
+	PreviousButtonStyle,
+	ButtonPanelStyle,
+	LeftActionStyle,
+	RightActionStyle,
+	WizardButton,
+} from './WizardFooterStyles';
+
+export default class WizardFooter extends Component<{
```

Comment:
I'm going to have to ask you to convert this to a functional component. We generally do not ever use classes anymore unless there is a specific use case like an ErrorBoundary.

Reply:
Done